src/components/common/CardHeader.vue

```diff
@@ -12,8 +21,8 @@ import { Component, Prop, Vue } from 'vue-property-decorator'
 
 @Component({})
 export default class CardHeader extends Vue {
-  @Prop({ required: true }) readonly icon!: string
-  @Prop({ required: true }) readonly label!: string
+  @Prop({ default: null }) readonly icon!: string
+  @Prop({ default: null }) readonly label!: string
```

Comment:
I updated this component to have optional icon (or label) so I could use it for this:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/bb83ced9-fa76-45ff-9239-86561ad386cc)